ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -213,3 +236,16 @@ def _stop(self):
     def stop(self):
         self._loop.call_soon_threadsafe(self._stop)
         self._ws_thread.join()
+
+    def get_successful_realizations(self):
+        mon = self.run()
+        for _ in mon.track():
+            pass
+        successful_reals = 0
+        for real in self._snapshot.to_dict()["reals"].values():
+            for stage in real["stages"].values():
+                # FIXME: assumes one stage per real
```

Comment:
How come?

Reply:
Because the legacy _is_ one stage and one step. The only thing the user can configure is the number and configuration of jobs.

There is no clear way to fix that `FIXME`. I can add a `TODO` and an issue if you want.

Reply:
Ok, so the point is that the code currently labels a realisation successful if any of the stages are deemed `Finished`? Can we assume that the length of `real["stages"] == 1`? Because is so, the `FIXME` could perhaps be turned into an `assert`?

Reply:
I can do that